src/cli_common.c

```diff
@@ -318,8 +318,8 @@ void parseRedisUri(const char *uri, const char* tool_name, cliConnInfo *connInfo
     UNUSED(tls_flag);
 #endif
 
-    const char *scheme = "redis://";
-    const char *tlsscheme = "rediss://";
+    const char *scheme = "valkey://";
+    const char *tlsscheme = "valkeys://";
```

Comment:
We need to support `redis:// and `rediss://` too for compatibility. Let's add code to support both, ok?

Reply:
Added follow-up issue #200 for testing this.